tests/Nodes/Shapes/SVGPathTest.php

```diff
@@ -51,6 +53,16 @@ public function testSetDescription()
         $this->assertSame(self::$sampleDescription, $obj->getAttribute('d'));
     }
 
+    public function testRasterizeWithNull()
+    {
+        $rast = $this->getMockBuilder('\SVG\Rasterization\SVGRasterizer')
+            ->disableOriginalConstructor()
+            ->getMock();
+        $obj = new SVGPath();
+
+        $this->assertNull($obj->rasterize($rast));
+    }
```

Comment:
Why is this test necessary? It seems a bit useless to me. (Since this test case is repeated multiple times, this comment applies to all of them.)